app/src/test/java/org/fossasia/openevent/app/presenter/EventDetailPresenterTest.java

```diff
@@ -130,7 +130,6 @@ public void shouldShowEventError() {
         inOrder.verify(eventDetailView).showProgressBar(true);
         inOrder.verify(eventRepository).getEvent(id, false);
         inOrder.verify(eventDetailView).showEventLoadError(error);
-        inOrder.verify(eventDetailView).showProgressBar(false);
```

Comment:
You just deleted a test for hiding the progress bar. Now anyone can remove the hiding code and be done with it because the tests will not fail. Please add a test which ensures that if someone forgets to hide the progress bar, it will fail and test it by forgetting to hide the progress bar and confirming that it fails